config/locales/en.yml

```diff
@@ -40,4 +40,8 @@ en:
         other: ">%{count}yr"
       almost_x_years:
         one:   "~1yr"
-        other: "~%{count}yr"
\ No newline at end of file
+        other: "~%{count}yr"
+  activerecord:
+    attributes:
+      agent:
+          receivers: Targets
```

Comment:
This feels like there's a naming problem, that the UI should match the model and show "Receivers", or that the model should be changed to `targets`. Otherwise the language gets confusing fast.